dali/pipeline/data/tensor_list_test.cc

```diff
@@ -73,12 +73,13 @@ class TensorListTest : public DALITest {
     }
 
     // Resize the buffer
-    tensor_list->Resize(shape);
+    tensor_list->Resize(shape, DALI_FLOAT);
 
     // Check the internals
-    ASSERT_NE(tensor_list->template mutable_data<float>(), nullptr);
+    ASSERT_TRUE(tensor_list->has_data());
```

Comment:
Shouldn't you specialize `has_data` to the TensorList now (as it is implemented in the buffer)?

Reply:
I maybe can do something, but I think it needs to wait for the proper changes.